cmd/tf-operator.v1/app/server.go

```diff
@@ -26,9 +26,9 @@ import (
 	"github.com/kubeflow/tf-operator/pkg/client/clientset/versioned/scheme"
 	tfjobinformers "github.com/kubeflow/tf-operator/pkg/client/informers/externalversions"
 	controller "github.com/kubeflow/tf-operator/pkg/controller.v1/tensorflow"
-	"github.com/kubeflow/tf-operator/pkg/util/signals"
+
+	"github.com/kubeflow/common/pkg/util/signals"
```

Comment:
Can we reorder the groups? We suggest placing third-party deps at the second group